docs/ProfileDeveloperGuide.md

```diff
@@ -490,6 +490,7 @@ Used by output serialisers where string output is required. `value` must be:
 * a string recognised by Java's `String.format` method
 * appropriate for the data type of `field`
 * not `null` (formatting will not be applied for null values)
+* formatting will not be applied if not applicable to the value
```

Comment:
This bullet point and the first bullet point are contradictory. This should be rephrased so that the behaviour is clear.